Fix aclocal warning:
/usr/share/aclocal/glib.m4:8: warning: underquoted definition of AM_PATH_GLIB
--- a/glib.m4
+++ b/glib.m4
@@ -7,3 +7,3 @@
 dnl
-AC_DEFUN(AM_PATH_GLIB,
+AC_DEFUN([AM_PATH_GLIB],
 [dnl 
